lychee-lib/src/types/error.rs

```diff
@@ -34,9 +34,12 @@ pub enum ErrorKind {
     /// The network client required for making requests cannot be created
     #[error("Error creating request client")]
     BuildRequestClient(#[source] reqwest::Error),
+    /// The Github client required for making requests cannot be created
+    #[error("Error creating Github client")]
+    BuildGithubClient(#[source] octocrab::Error),
     /// Network error while using Github API
     #[error("Network error (GitHub client)")]
-    GithubRequest(#[from] octocrab::Error),
+    GithubRequest(#[source] octocrab::Error),
```

Comment:
why is `from` replaced by `source`?

Reply:
That was a mistake. Nice catch.
During refactoring I had a case where I copied this error and converted it to `BuildGithubClient`. Then I had to `from` impls and I accidentially removed it from both. Rerverted now.